### ISSUE TITLE
Tags corresponding to official release tarballs

### ISSUE THREAD

**Issue by ghisvail** [pocl/pocl#316]
Hi, could you consider pushing tags corresponding to the specific commit used for generating the release tarballs on the project's website. At least for the current stable release.

Thanks.

**Comment by kwm81** [comment]
This would be very usefull to have for downstream. +1

**Comment by ghisvail** [comment]
@kwm81 Do you guys have `pocl` successfully ported to FreeBSD?

**Comment by kraiskil** [comment]
I see no reason why not. Let's add a note on this to the releasing doc?
(and last I checked, pocl did work on FreeBSD.)

**Comment by ghisvail** [comment]
Awesome, this way I can synchronize the Debian packaging with your repository. Right now, I am working on a snapshot based on `master` but would rather track tagged releases in the future.

**Comment by kwm81** [comment]
Yes, I have a WIP port of pocl. It only needs some polish and I need to rewrite a patch so it can be upstreamed.

**Comment by ghisvail** [comment]
Thanks for pushing a tag for the current version `0.13`. Please try to keep a consistent naming convention for them in the future, as they are used downstream to [track](https://wiki.debian.org/debian/watch) new releases. Previous tags were prefixed with a `v`, whilst the new one is not.

I am also questioning the usefulness of adding a `RC\d` suffix considering the library has yet to hit `1.0` anyway. This suffix can also gets in the way in case we (downstream) want to test a snapshot release, for which a special tag is created by appending the date and hash of the commit to the latest release tag. Obviously, the call is yours in the end but I thought I should raise this now.

**Comment by franz** [comment]
OK, i think this bug is resolved. Creating tags is now in the official release procedure.
